nncf/common/quantization/structs.py

```diff
@@ -89,6 +89,16 @@ def is_a_bitwidth_variant(self, other_qconfig: 'QuantizerConfig') -> bool:
                self.signedness_to_force == other_qconfig.signedness_to_force and \
                self.mode == other_qconfig.mode
 
+    def get_state(self) -> Dict:
+        return {'num_bits': self.num_bits,
+                'mode': self.mode,
+                'signedness_to_force': self.signedness_to_force,
+                'per_channel': self.per_channel}
```

Comment:
It looks like that `narrow_range` and `half_range` should be also stored in the state to definitely restore quantization parameters.

Reply:
Currently it's not part of `QuantizerConfig`. I think we should extend it first and then add to state

Reply:
@daniil-lyakhov @aleksei-kashapov 
Is the model with saturation fix loaded correctly on develop? Is there test for it?

Reply:
@ljaljushkin @alexsu52 The models should be loaded correctly. There is no specific test, but saturation fix applies for given target devices, e.g. CPU, which is the default for many tests that we have. So for all of these tests, this fix should apply. In other words, if we have a test checking loading model for CPU, it is already should load the model with saturation fix.
Does it sound okay for you?

Reply:
I think it's not enough, if you're speaking about just loading weights without errors.
There should be some check that parameters of quantization particularly for saturation fix (narrow_range and half_range) can be restored correctly, instead of initialization by default for instance in resume mode.

Do we have such test?

Reply:
No, we don't.
I take care of it

Reply:
Created task 58309 for the test and 58312 for extending `QuantizerConfig`

Reply:
@ljaljushkin Please, confirm that checkpoints trained with saturation fix will be resumed correctly.

Reply:
https://github.com/openvinotoolkit/nncf/pull/791 Add appropriate test

Reply:
Here's the [test](https://github.com/openvinotoolkit/nncf/pull/682/files#diff-ce170264cee4bc737448b6df49bd12c9d7f7f9f2546dc96c1528b05eebe90586R399-R410) checking that checkpoints trained with saturation fix will be resumed and exported correctly.
It works even now, despite half_range is not saved in the state. It happens because saturation fix is enabled by config, which is mandatory now.